api/src/main/java/com/nicknathanjustin/streamercontracts/votes/VoteServiceImpl.java

```diff
@@ -144,8 +142,8 @@ private void validateContract(@NonNull final ContractModel contractModel) {
             throw new IllegalArgumentException("Voting logic not yet implemented for community contracts. ContractId " + contractModel.getId());
         }
 
-        if (!contractModel.isAccepted()) {
-            throw new IllegalArgumentException("Cannot vote on an unaccepted contract. ContractId " + contractModel.getId());
+        if (!contractModel.isAccepted() && !isContractExpired(contractModel)) {
+            throw new IllegalArgumentException("Cannot vote on an unaccepted contract that hasnt expired yet. ContractId " + contractModel.getId());
```

Comment:
After the contract has expired, you can vote on an unaccepted contract?

Reply:
Good catch. The exception message was off. Should change 'vote' to 'settle'. Settle being payout funds. Fixed.